compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/syntax/tree/SyntaxTree.java

```diff
@@ -124,6 +124,17 @@ public String toSourceCode() {
         return rootNode.toSourceCode();
     }
 
+    /**
```

Comment:
Shall we add this method inside the formatter without altering the parser?

Reply:
Moved.